slides/src/content/components.md

```diff
@@ -312,6 +312,75 @@ class ChildComponent {}
 
 ---
 
+## Template Projection (2/3)
+
+- Multiple `<ng-content>` tags can be used in a template by using the `select` attribute
+- In the template, we can use an HTML tag, say, `<header>` to specify the position of projected content to the `ng-content` with `select="header"`
+
+```ts
+@Component({
+  selector: 'rio-app',
+  template: `<rio-child>
+              <header>
+               <p>This is my header content</p>
+              </header>
+              <footer>
+                <p>This is my footer content</p>
+              </footer>
+             </rio-child>`
+})
+class AppComponent {}
+```
+
+```ts
+@Component({
+  selector: 'rio-child',
+  template: `<h4>Child Component</h4>
+             <ng-content select="header"></ng-content>
+             <ng-content select="footer"></ng-content>`
+})
+class ChildComponent {}
+```
+
+[View Example](https://plnkr.co/edit/kwD3iKLU8mELAoHj2fBv?p=preview)
```

Comment:
We have too much code (or text) in this slide, it's not rendering correctly in my laptop

<img width="1440" alt="screen shot 2017-02-01 at 5 01 55 pm" src="https://cloud.githubusercontent.com/assets/2279046/22528546/d195e734-e8a1-11e6-8c6c-58cbe4d70071.png">